src/types/types.ts

```diff
@@ -1348,6 +1344,11 @@ export interface EdgeSwapQuote {
    * Maximum amount of time this quote will take to be fulfilled (in seconds)
    */
   readonly maxFulfillmentSeconds?: number
+
+  /** Worse case amount - the highest cost for a 'from' quote, and the lowest
+   * receive amount for a 'to' quote */
+  readonly worstAmount?: string
```

Comment:
Change to minAmount. Regardless of the quote being `from` or `to`, this will always be the minimum RECEIVE amount. I'd be ok renaming to `minReceiveAmount`